packages/matchbox/src/components/OptionalLabel/OptionalLabel.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Box } from '../Box';
+
+const StyledBox = styled(Box)`
+  float: right;
```

Comment:
I *think* its only supposed to be on the right in TextField and Select, but for checkbox/radio groups itll just be next to the legend. I'd ask Aubrey